Makefile

```diff
@@ -21,11 +21,11 @@ nadiff: $(obj)
 
 install: nadiff
 > mkdir -p ${PREFIX}/bin
-> cp -f nadiff ${PREFIX}/bin
+> cp -f nadiff git-nadiff ${PREFIX}/bin
 > chmod 755 ${PREFIX}/bin/nadiff
```

Comment:
Sorry, I forgot to add that we also need this for git-nadiff 
chmod 755 ${PREFIX}/bin/git-nadiff